executor/executor.go

```diff
@@ -189,8 +188,8 @@ func filterTeardowns(build screwdriver.Build) ([]screwdriver.CommandDef, []screw
 	userTeardownCommands := []screwdriver.CommandDef{}
 
 	for _, cmd := range build.Commands {
-		isSdTeardown, _ := regexp.MatchString("^sd-teardown-*", cmd.Name)
-		isUserTeardown, _ := regexp.MatchString("^teardown-*", cmd.Name)
+		isSdTeardown, _ := regexp.MatchString("^sd-teardown-.*", cmd.Name)
```

Comment:
can we change these to `^sd-teardown-.+`

Reply:
Makes sense. so that `teardown-` shouldn't match?

Reply:
yes

Reply:
fixed!